moe/views/utils.py

```diff
@@ -123,13 +150,24 @@ def _make_gp_from_params(params):
                 point['value_var'],
             )
         )
-
-    gaussian_process = GaussianProcess(
+    optimizer_type = None
```

Comment:
also, just a tip: python prefers ``my_key in my_dict`` over ``my_dict.has_key(my_key)``:
https://docs.python.org/2/library/stdtypes.html#dict.has_key
deprecated as of python 2.3 and removed in python 3.